src/libostree/ostree-repo.c

```diff
@@ -4611,11 +4618,21 @@ ostree_repo_regenerate_summary (OstreeRepo     *self,
       if (!ostree_repo_load_variant (self, OSTREE_OBJECT_TYPE_COMMIT, commit, &commit_obj, error))
         goto out;
 
-      g_variant_builder_add_value (refs_builder, 
+      g_variant_dict_init (&commit_metadata_builder, NULL);
+
+      /* Forward the commit’s timestamp if it’s valid. */
+      commit_timestamp = ostree_commit_get_timestamp (commit_obj);
+      dt = g_date_time_new_from_unix_utc (commit_timestamp);
+
+      if (commit_timestamp > 0 && dt != NULL)
```

Comment:
I was treating `commit_timestamp == 0` as unset in general, but I guess that probably doesn’t apply here. Commits should always have timestamps. I’ll drop that clause.